mvc/src/main/java/com/interface21/webmvc/servlet/view/JspView.java

```diff
@@ -14,18 +15,25 @@ public class JspView implements View {
 
     public static final String REDIRECT_PREFIX = "redirect:";
 
-    public JspView(final String viewName) {
+    private final String viewName;
+
+    public JspView(String viewName) {
+        this.viewName = viewName;
     }
 
     @Override
-    public void render(final Map<String, ?> model, final HttpServletRequest request, final HttpServletResponse response) throws Exception {
-        // todo
+    public void render(Map<String, ?> model, HttpServletRequest request, final HttpServletResponse response) throws Exception {
```

Comment:
```suggestion
    public void render(Map<String, ?> model, HttpServletRequest request, HttpServletResponse response) throws Exception {
```
미처 없애지 못한 final이 존재하네요 ㅎㅎ.

Reply:
수정이 안된 것 같아요~

Reply:
푸시를 깜빡했네요 ㅜㅜ 죄송합니다